npm-client/src/cancel_order.ts

```diff
@@ -115,3 +116,66 @@ export async function cancelOrdersForMarket(
   });
   return response.body;
 }
+
+/**
+ * For the provided order and market position calculate amount that will be refunded if order gets canceled.
+
+ * @param order {Order} order to be canceled
+ * @param marketPosition {MarketPosition} market position of the order's owner
+ * @returns the amount of the refund in raw form; this means it needs to be divided by the mint decimals before it can be dispalyed
+ *
+ * @example
+ *
+ * const marketPk = new PublicKey('7o1PXyYZtBBDFZf9cEhHopn2C9R4G6GaPwFAxaNWM33D')
+ * const cancelledOrders = await cancelOrdersForMarket(program, marketPk)
+ */
+export function cancelOrderRefund(
```

Comment:
I'd maybe suggest renaming this function. The other functions that start with `cancel...` actually trigger an on chain instruction, whereas this is just getting/calculating/estimating a value.

Reply:
`calculateOrderCancellationRefund` ?